app/src/main/java/com/android/sample/ui/map/MapScreen.kt

```diff
@@ -163,7 +163,7 @@ fun MapScreen(
               modifier =
                   Modifier.fillMaxWidth()
                       .height(76.dp)
-                      .background(color = Purple80)
+                      .background(color = LightPurple)
```

Comment:
It might be beneficial to define this color within the app's light and dark MaterialTheme.colorScheme.